google-cloud-datastore/pom.xml

```diff
@@ -38,12 +38,17 @@
       <version>1.3.0</version>
       <exclusions>
         <exclusion>
-          <artifactId>guava-jdk5</artifactId>
           <groupId>com.google.guava</groupId>
+          <artifactId>guava-jdk5</artifactId>
         </exclusion>
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-common-protos</artifactId>
+      <version>${generatedProto.version}</version>
+    </dependency>
```

Comment:
This doesn't work. From a combination of experimenting and reading the manual, I think the fundamental reason is that dependency exclusion only excludes packages from the classpath, but version resolution still needs to work out. I think @vam-google knows Maven a lot more than I do, could you fact-check? 

The "right way" is for datastore to depend on `proto-google-common-protos`, but since it depends on `grpc`, I think we're still forced to override the dependency version.